### PR TITLE
collector: Move collector admin endpoints

### PR DESCRIPTION
**Pull request by franklinhu**
The methods for modifying the sample rate and storage request rate don't really belong in the collector's thrift interface. Moved them to the ostrich admin interface.

### CODE REVIEW

File: zipkin-collector-core/src/main/scala/com/twitter/zipkin/config/ZipkinCollectorConfig.scala [hunk 1]
```diff
@@ -106,8 +107,58 @@ trait ZipkinCollectorConfig extends ZipkinConfig[ZipkinCollector] {
...
+    addConfigEndpoint()
     new ZipkinCollector(this)
   }
+
+  /**
+   * Add endpoints to the Ostrich admin service for configuring the adjustable values
+   *
+   * Methods:
+   *   /config/get/sampleRate
+   *   /config/set/sampleRate?value=0.1
```
Comment (johanoskarsson) [inline zipkin-collector-core/src/main/scala/com/twitter/zipkin/config/ZipkinCollectorConfig.scala]: I assume Ostrich doesn't support POST?

File: zipkin-collector-core/src/main/scala/com/twitter/zipkin/config/ZipkinCollectorConfig.scala [hunk 2]
```diff
@@ -106,8 +107,58 @@ trait ZipkinCollectorConfig extends ZipkinConfig[ZipkinCollector] {
   val serverConfig: CollectorServerConfig
 
   def apply(runtime: RuntimeEnvironment): ZipkinCollector = {
+    addConfigEndpoint()
```
Comment (johanoskarsson) [inline zipkin-collector-core/src/main/scala/com/twitter/zipkin/config/ZipkinCollectorConfig.scala:110]: Should probably add some tests for this

Comment (franklinhu) [inline zipkin-collector-core/src/main/scala/com/twitter/zipkin/config/ZipkinCollectorConfig.scala:110]: Done

**Comment by johanoskarsson** [comment]
+1

**Pull request closed without merging**